src/triage/util/db.py

```diff
@@ -29,4 +50,4 @@ def __reconstruct__(cls, url, creator, kwargs):
         return cls(url, creator=creator, **kwargs)
 
 
-create_engine = SerializableDbEngine
+create_engine = functools.partial(SerializableDbEngine, json_serializer=json_dumps)
```

Comment:
On second look, it seems like this wouldn't interfere at all with checking that we receive a `SerializableDbEngine`, for example. That is, say, this should still work:

    if isinstance(db_engine, SerializableDbEngine):
        …

Reply:
(Of course, I don't know that we have to override `create_engine`.)